proto/interchain_security/ccv/provider/v1/tx.proto

```diff
@@ -59,3 +62,19 @@ message MsgSubmitConsumerMisbehaviour {
 }
 
 message MsgSubmitConsumerMisbehaviourResponse {}
+
+
+// MsgSubmitConsumerDoubleVoting defines a message that reports an equivocation
+// observed on a consumer chain
+message MsgSubmitConsumerDoubleVoting {
```

Comment:
This definition stays unchanged bc the Hermes relayer is supposedly already using it.